apps/judicial-system/web/src/components/BlueBoxWithIcon/BlueBoxWithDate.tsx

```diff
@@ -0,0 +1,291 @@
+import { FC, useContext, useEffect, useState } from 'react'
+import { useIntl } from 'react-intl'
+import addDays from 'date-fns/addDays'
+import { AnimatePresence, motion } from 'framer-motion'
+
+import {
+  Box,
+  Button,
+  Icon,
+  IconMapIcon,
+  Text,
+  toast,
+} from '@island.is/island-ui/core'
+import { formatDate } from '@island.is/judicial-system/formatters'
+import { VERDICT_APPEAL_WINDOW_DAYS } from '@island.is/judicial-system/types'
+import { errors } from '@island.is/judicial-system-web/messages'
+
+import {
+  CaseIndictmentRulingDecision,
+  Defendant,
+  ServiceRequirement,
+} from '../../graphql/schema'
+import { formatDateForServer, useDefendants } from '../../utils/hooks'
+import DateTime from '../DateTime/DateTime'
+import { FormContext } from '../FormProvider/FormProvider'
+import { getAppealExpirationInfo } from '../InfoCard/DefendantInfo/DefendantInfo'
+import SectionHeading from '../SectionHeading/SectionHeading'
+import { strings } from './BlueBoxWithDate.strings'
+import * as styles from './BlueBoxWithIcon.css'
+
+interface Props {
+  defendant: Defendant
+  indictmentRulingDecision?: CaseIndictmentRulingDecision | null
+  icon?: IconMapIcon
+}
+
+type DateType = 'verdictViewDate' | 'appealDate'
+
+const BlueBoxWithDate: FC<Props> = (props) => {
```

Comment:
I feel like this component is very specific to verdict view datss and appeal dates which isn't that apparent from its title/location in the projects folder structure. I would assume if I was coming in to the project that it would be a reusable component for any date I'd like, which it isn't. I think it's fine to leave it like this to get it out but maybe in the future we could either move it closer to where it's used, or make it more generalized and reusable and keep it in the components folder.

Reply:
I agree. This is a product of us running super fast rn. I think we should leave it as is for now, but let's be super quick to react if this causes any confusion in the future.